Formula/aws-sdk-cpp.rb

```diff
@@ -0,0 +1,42 @@
+class AwsSdkCpp < Formula
+  desc "AWS SDK for C++"
+  homepage "https://github.com/aws/aws-sdk-cpp"
+  url "https://github.com/aws/aws-sdk-cpp/archive/0.10.6.tar.gz"
+  sha256 "01ad50677e2bde3bcd51128a9e043f76371e9642a5bea235637ce8e450eec5de"
+  head "https://github.com/aws/aws-sdk-cpp.git"
+
+  option "with-custom-memory-management", "Build with custom memory management"
+  option "with-static", "Build with static linking"
+  option "without-http-client", "Don't include the libcurl HTTP client"
```

Comment:
When static linking... If you don't enable this, you end up being required to also link in libcurl which can cause problems.